test/clt-tests/sharding/test-drop-sharding-table.rec

```diff
@@ -0,0 +1,82 @@
+| %{NUMBER} | 127.0.0.1:1312 | CREATE TABLE IF NOT EXISTS `tbl1_s0` (id bigint)                                                                            |           %{NUMBER} |     %{NUMBER} | processed | %{NUMBER} | %{NUMBER} |        %{NUMBER} |
+| %{NUMBER} | 127.0.0.1:1312 | CREATE TABLE IF NOT EXISTS `tbl1_s2` (id bigint)                                                                            |           %{NUMBER} |     %{NUMBER} | processed | %{NUMBER} | %{NUMBER} |        %{NUMBER} |
+| %{NUMBER} | 127.0.0.1:2312 | CREATE TABLE IF NOT EXISTS `tbl1_s1` (id bigint)                                                                            |           %{NUMBER} |     %{NUMBER} | processed | %{NUMBER} | %{NUMBER} |        %{NUMBER} |
+| %{NUMBER} | 127.0.0.1:1312 | CREATE TABLE `tbl1`
+type='distributed' local='tbl1_s0' local='tbl1_s2' agent='127.0.0.1:2312:tbl1_s1'
+|           %{NUMBER} |     %{NUMBER} | processed | %{NUMBER} | %{NUMBER} |        %{NUMBER} |
+| %{NUMBER} | 127.0.0.1:2312 | CREATE TABLE `tbl1`
+type='distributed' local='tbl1_s1' agent='127.0.0.1:1312:tbl1_s0' agent='127.0.0.1:1312:tbl1_s2'
+|           %{NUMBER} |     %{NUMBER} | processed | %{NUMBER} | %{NUMBER} |        %{NUMBER} |
+| %{NUMBER} | 127.0.0.1:2312 | DROP TABLE IF EXISTS tbl1                                                                                                   |           %{NUMBER} |     %{NUMBER} | processed | %{NUMBER} | %{NUMBER} |        %{NUMBER} |
+| %{NUMBER} | 127.0.0.1:2312 | DROP TABLE IF EXISTS tbl1_s1                                                                                                |           %{NUMBER} |     %{NUMBER} | processed | %{NUMBER} | %{NUMBER} |        %{NUMBER} |
+| %{NUMBER} | 127.0.0.1:1312 | DROP TABLE IF EXISTS tbl1                                                                                                   |           %{NUMBER} |     %{NUMBER} | processed | %{NUMBER} | %{NUMBER} |        %{NUMBER} |
+| %{NUMBER} | 127.0.0.1:1312 | DROP TABLE IF EXISTS tbl1_s0                                                                                                |           %{NUMBER} |     %{NUMBER} | processed | %{NUMBER} | %{NUMBER} |        %{NUMBER} |
+| %{NUMBER} | 127.0.0.1:1312 | DROP TABLE IF EXISTS tbl1_s2                                                                                                |           %{NUMBER} |     %{NUMBER} | processed | %{NUMBER} | %{NUMBER} |        %{NUMBER} |
++----------------+----------------+-----------------------------------------------------------------------------------------------------------------------------+-------------+-------+-----------+---------------+---------------+----------+
+––– input –––
+cat /var/log/manticore-1/searchd.log
+––– output –––
```

Comment:
I'm not sure that we really need this log and a lot of regexps here, probably we just can remove this check or look for something exactly what we want – for example, `becoming master`

Reply:
Fixed